backend/src/main/java/gov/cdc/usds/simplereport/validators/FileValidator.java

```diff
@@ -30,10 +30,10 @@ public List<FeedbackMessage> validate(InputStream csvStream) {
 
     if (!valueIterator.hasNext()) {
       var feedback =
-          new FeedbackMessage(
-              CsvValidatorUtils.ITEM_SCOPE,
-              "File is missing headers and other required data",
-              null);
+          FeedbackMessage.builder()
+              .scope(CsvValidatorUtils.ITEM_SCOPE)
```

Comment:
I know this wasn't your change, but would this be more of a "report" scope if there may be missing headers that would affect all rows?

Reply:
I agree, honestly, I think we should remove the scope all together, the UI doesn't use it

Reply:
Even better!

Reply:
Am I understanding it correctly that currently the `ITEM_SCOPE` specifies an issue with an individual row as opposed to an issue with the entire file like a `REPORT_SCOPE`? I agree it seems good to remove scope if it's not being used especially since we're having more specific error messages that might make scope redundant